Makefile

```diff
@@ -53,3 +54,6 @@ install:
 uninstall:
 	/bin/rm -f $(DESTDIR)/$(MODULE_DESTDIR)/nv_peer_mem.ko
 	if [ ! -n "$(DESTDIR)" ]; then $(DEPMOD) -r -ae $(KVER);fi;
+
+gen_nv_symvers:
+	$(PWD)/create_nv.symvers.sh $(KVER)
```

Comment:
@haggaie no, it should be KVER.
actually KERNEL_VER seems like a leftover.... it's not used or set anywhere else.